Article/Article.js

```diff
@@ -1,26 +1,38 @@
 // Start at `let articles`
 
 class Article {
-  constructor(element) {
-    // assign this.element to the passed in article element
-    this.element;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference (or article element), calling the expandArticle method.
-    
+  constructor(article) {
+    this.article = article;
+    this.expandButton = article.querySelector('.expandButton')
+    this.expandButton.innerText = 'expand';
+    this.expandButton.addEventListener('click', () => this.expandArticle());
+    this.closeButton = article.querySelector('.closeButton');
+    this.closeButton.addEventListener('click', () => this.closeArticle());
+    console.log(this.closeButton)
+
   }
 
   expandArticle() {
     // Using our reference to the article element, add or remove a class to expand or hide the article.
+    this.article.classList.toggle('article-open')
+    TweenMax.fromTo([this.article], 1.5, {
```

Comment:
To get the close animation to work, you would need to wrap this open animation in an if statement that checks whether the article is already opened by checking the classList property